### ISSUE TITLE
Fandango ingestion from the Articles tools appear as annotated and don't have 'About', 'Mentions', 'Topic'

### ISSUE THREAD

**Issue by vittorianovancini** [fandangOrg/fandango#109]
All news analyzed by Fandango Articles Tool appear in Dashbord without any value in the fields ABOUT, MENTIONS and TOPIC - an "processType" is "Annotated" as for the Annotations Articles. 
![image](https://user-images.githubusercontent.com/37619775/108828355-06535b80-75c7-11eb-91f5-29292fd22666.png)
I'm sure (I wish) this is a quick one to fix... Ciao. Vittoriano

**Comment by pstalidis** [comment]
This is intentional.

The topic and about fields will be filled **after** the ingestion

**Comment by mmagaldi-eng** [comment]
As far as I know, @vittorianovancini noted an issue about the online process: all last articles processed online (using the UI) are erroneously handled (and marked) as "annotated" data. 

@vittorianovancini, is it right?

**Comment by vittorianovancini** [comment]
Yes, and I've not seen any reprocessing on the articles I submitted by Fandango's tool. When is that supposed to happen? The day after, when system is switched on again?

**Comment by macagari** [comment]
The issue Vittoriano reported is correct, I changed it through the k8s pod, it should work but I'm checking now the only problem is that the online process is not working, I'll check it out. Anyway, I'm going to push another image with some UI changes Vittoriano suggested last week so it will be as usual, annotated as "online". Thank you for reporting that!

**Comment by mmagaldi-eng** [comment]
The _about_ field now is filled, _Topic_ and _Mentions_ are still empty fields.   

@pstalidis, please note that half of the _about_ keywords in Italian are stopwords.
